### ISSUE TITLE
Download single annotation dialogs - step 1 of 2

### ISSUE THREAD

**Issue by junxini** [chanzuckerberg/cryoet-data-portal#435]
**Description**: allow users to download a single annotation through web interface

**Tasks:**
- [ ] **Download dialog - step 1 of 2**
 - dialog header space to include "Step 1 of 2", the standard "Configure Download" title, and subtext shown in the gray ( dataset, run, annotation ID, annotation object (?TBC?), object shape type) 
 - allow user to select file format for download: 
   - if there are multiple file formats, default to MRC
   - if there's only one file format available, default to it (without requiring user to select before clicking on the next button)
 - info icon next to "File Format" will bring up a tool tip and link to documentation
   - dependent on #466
 - clicking on "Next" will bring user to step 2 of 2
- [ ] **Download dialog - step 2 of 2** refer to #450

**Link to design:** https://www.figma.com/file/gwX2PjfGPfMrbEWMMJq6rW/H1-2024---Annotations-Table-Update?node-id=527%3A139630&mode=dev

**Comment by kne42** [comment]
note to self: example query
```
  annotations_by_pk(id: 112) {
    files(where: {shape_type: {_eq: "Point"}}) {
      format
      id
      s3_path
      https_path
      shape_type
    }
  }
```

**Comment by manasaV3** [comment]
The value for `id ` in `annotation_by_pk` should be populated by value from the `annotation id` column of the table. The value for shape_type is from `object shape type` column in the table.

**Comment by junxini** [comment]
@kne42 FYI Dannielle has closed out the tool tip copy dependency in #466
